### ISSUE TITLE
Add items without ratings

### ISSUE THREAD

**Issue by nliautaud** [mukurokudo/imdb2senscritique#5]
It would be great to "check" movies without ratings.
The api seems to be `reviews/add` with `is_done=1` and `date_done=?`  :
```
/sc2/reviews/add/SCMOVIEID.json?format_id=0&is_recommend=0&is_current=0&is_done=1&is_wish_list=0&is_shopping_list=0&rating=&review_content_source=&review_content=&review_title=&review_state=1&review_comment_privacy=0&date_done=&productId=SCMOVIEID
```

**Comment by mukurokudo** [comment]
What do you mean by checking movies without rating? Filtering the searched title list so only the unrated one come up? 
Anyway, we aware that any curl call to Sc will diminish drastically the script performances .

**Comment by nliautaud** [comment]
Nope, I was thinking about adding support to CSV records with an empty rating field. Instead of rating the entry, it could be "checked as viewed" on SC using the api above. It wouldn't be an additional call, just an increase of supported entries.
I have for example a lot of items "checked" on trakt.tv, but only a part of them are rated. SC offer the same feature, even if it's hidden trough their website UI.

(On peut se parler en français, j'imagine ?)
